### PR TITLE
style(submit-channel): improve submit channel design

### PR DESCRIPTION
**Pull request by JimmyMow**
A few simple CSS tweaks to design our submit channel form per it's design mocks

### CODE REVIEW

File: app/components/Contacts/SubmitChannelForm.scss [hunk 1]
```diff
@@ -2,17 +2,24 @@
...
-  font-family: Roboto;
+  font-family: Roboto, sans-serif;
   color: $white;
   margin: 0 auto;
   width: 500px;
 
   .header {
     padding: 20px;
+    text-align: center;
+    padding-bottom: 20px;
```
Comment (Empact) [inline app/components/Contacts/SubmitChannelForm.scss]: redundant with `padding` setting

File: app/components/Contacts/SubmitChannelForm.scss [hunk 2]
```diff
@@ -2,17 +2,24 @@
...
+    border-bottom: 1px solid $spaceborder;
 
     h1 {
       margin-bottom: 15px;
       font-size: 20px;
+      font-size: 22px;
```
Comment (Empact) [inline app/components/Contacts/SubmitChannelForm.scss]: duplicate key `font-size`

File: app/components/Contacts/SubmitChannelForm.scss [hunk 3]
```diff
@@ -2,17 +2,24 @@
...
+      font-weight: 100;
+      margin-top: 10px;
```
Comment (Empact) [inline app/components/Contacts/SubmitChannelForm.scss]: Maybe combine this with `margin-bottom`, above?

**Comment by JimmyMow** [comment]
@Empact updated based on comments

File: app/components/Contacts/SubmitChannelForm.scss [hunk 4]
```diff
@@ -2,17 +2,21 @@
...
   .header {
-    padding: 20px;
+    padding: 20px 0;
+    text-align: center;
+    border-bottom: 1px solid $spaceborder;
 
     h1 {
-      margin-bottom: 15px;
-      font-size: 20px;
+      margin-bottom: 10px 0 15px 0;
```
Comment (Empact) [inline app/components/Contacts/SubmitChannelForm.scss]: This should be `margin`

**Comment by JimmyMow** [comment]
@Empact updated

**Comment by Empact** [comment]
utACK 7e426eb

**Pull request merged**